lib/rbs/prototype/rb.rb

```diff
@@ -166,9 +178,9 @@ def process(node, decls:, comments:, singleton:)
             end
           when :attr_reader
             args.each do |arg|
-              if arg&.type == :LIT && arg.children[0].is_a?(Symbol)
+              if arg && (name = literal_to_symbol(arg))
```

Comment:
It is not the main part of this pull request, but `attr_*` accept Strings as a method names. It is the same behavior with `alias_method`.
So I changed them to accept Strings.


```ruby
class C
  attr_reader 'a'
  attr_writer 'a'
  attr_accessor 'b'
end

c = C.new

c.a = 1
c.b = 2
p c.a, c.b
# => 1
     2
```

https://github.com/ruby/rbs/pull/457/commits/f8accb0b82865f63ad0266c66fdee4f3974f0b8b